examples/opencv_rc.py

```diff
@@ -0,0 +1,541 @@
+        # any complex calculations and send motor commands
+        if self._action['linear'] != linear or \
+                self._action['angular'] != angular:
+
+            # Set the value of the new action
+            self._action['linear'] = linear
+            self._action['angular'] = angular
+
+            # If Cozmo is not doing anything, simply stop all the motors
+            if self._action['linear'] == Direction.NONE and \
+                    self._action['angular'] == Direction.NONE:
+                self._cozmo_clt.stop_all_motors()
+
+            # Compute the speed of each wheel depending on Cozmo's action and
+            # the new values for the linear and angular velocities
+            else:
+                # Are we going forward, backward, or not moving in this
+                # direction at all?
+                if self._action['linear'] == Direction.FORWARD:
+                    lin_vel = self._velocity['linear']
+                elif self._action['linear'] == Direction.BACKWARD:
+                    lin_vel = -self._velocity['linear']
+                else:
+                    lin_vel = 0
+
+                # Are we going left, right, or not moving in this direction at
+                # all?
+                if self._action['angular'] == Direction.LEFT:
+                    ang_vel = self._velocity['angular']
+                elif self._action['angular'] == Direction.RIGHT:
+                    ang_vel = -self._velocity['angular']
+                else:
+                    ang_vel = 0
+
+                # Compute the actual speed of each wheel
+                left = min(pc.MAX_WHEEL_SPEED.mmps, lin_vel -
+                           (pc.TRACK_WIDTH.mm * ang_vel) / 2)
+                right = min(pc.MAX_WHEEL_SPEED.mmps, lin_vel +
+                            (pc.TRACK_WIDTH.mm * ang_vel) / 2)
+
+                # Send the command to the motors
+                self._cozmo_clt.drive_wheels(lwheel_speed=left,
+                                             rwheel_speed=right)
+
+    def _stop_all(self, *args):
+        """
+        Simply stop all motors and set both linear and angular actions to NONE.
+        This is a wrapper for ease of use as a callback.
+        :return: None
+        """
+
+        # Set both actions to NONE, which will stop the motors
+        self._set_action(Direction.NONE, Direction.NONE)
+
+    def _on_head_tilt_change(self, value):
+        """
+        Simply change the head tilt based on the value given in parameter.
+        :param value: Float. A value between 0: head fully down,
+        and 100: head fully up.
+        :return: None
+        """
+
+        # Transform the value into a percentage
+        value /= 100
+
+        # Set the new head tilt based on the value in parameter
+        self.head_tilt = value * pc.MAX_HEAD_ANGLE.radians + \
+            (1 - value) * pc.MIN_HEAD_ANGLE.radians
+
+    def _on_lift_height_change(self, value):
```

Comment:
Same as for the head tilt.

Reply:
The same comment as above applies here. I am using a property setter to make sure  the new value stays within Cozmo's physical constraints and send a command for Cozmo to move its lift to new heights. The code corresponding to the property setter is located on line 469.